android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/BluetoothSite.java

```diff
@@ -228,7 +228,16 @@ private File getHtmlFile() {
     return new File(websiteDir, "website.html");
   }
 
+  private File getTempFile() {
+    File websiteDir = new File(activity.getFilesDir(), "Websites");
+    websiteDir.mkdir();
+    return new File(websiteDir, "temp.html");
```

Comment:
trying to implement the File.createTempFile() and I keep getting error
java.lang.IllegalArgumentException: Failed to find configured root that contains /data/data/physical_web.org.physicalweb/cache/temp-72855774.tmp